cli/js/lib.deno.ns.d.ts

```diff
@@ -16,7 +16,12 @@ declare namespace Deno {
     fn: () => void | Promise<void>;
     name: string;
     ignore?: boolean;
+    /** Check that the number of async completed ops after the test is the same
+     * as number of dispatched ops. Defaults to false.*/
     sanitizeOps?: boolean;
+    /** Ensure the test case does not "leak" resources - ie. the resource table
+     * after the test has exactly the same contents as before the test. Defaults
+     * to true. */
```

Comment:
There might have been an error in an issue; we meant to add a comment that these values are `true` by default

Reply:
OK, so leave the existing default values (true) and just update the docs then?

Reply:
Yes 👍